landing-page/components/NewsLetterForm.tsx

```diff
@@ -0,0 +1,180 @@
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import { tw } from '@helpwave/common/twind'
+import { Span } from '@helpwave/common/components/Span'
+import { Input } from '@helpwave/common/components/user-input/Input'
+import { noop } from '@helpwave/common/util/noop'
+import { useState } from 'react'
+import { Select } from '@helpwave/common/components/user-input/Select'
+import { Button } from '@helpwave/common/components/Button'
+
+const positionList = ['employee', 'manager', 'investor', 'medical_personal'] as const
+export type Position = typeof positionList[number]
```

Comment:
Let us ask after the industry of the company rather than the position in the company.

Example industries: `'Investment', 'Hospital', 'Outpatient Care', 'Research', 'Development'`